tests/external_tests.rs

```diff
@@ -0,0 +1,12 @@
+mod helpers;
+
+use helpers::in_directory as cwd;
+
+#[test]
+fn external_command() {
+    // Echo should exist on all currently supported platforms. A better approach might
+    // be to generate a dummy executable as part of the tests with known semantics.
+    nu!(output, cwd("tests/fixtures"), "echo 1");
```

Comment:
Should this be ^echo?

Reply:
I meant to have two tests -- one `echo` and one `^echo`. Good catch.

Reply:
Let's land this. I'll work on the idea listed in the comment so you guys can keep working on the other things.

Reply:
@androbtech done